devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy

```diff
@@ -14,12 +14,15 @@
 metadata {
 	definition (name: "Z-Wave Switch Generic", namespace: "smartthings", author: "SmartThings") {
 		capability "Actuator"
+		capability "Health Check"
  		capability "Switch"
 		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
 
 		fingerprint inClusters: "0x25", deviceJoinName: "Z-Wave Switch"
+		fingerprint mfr:"001D", prod:"1A02", model:"0334", deviceJoinName: "Leviton DZPA1 Plug-in Appliance Module"
```

Comment:
Good idea, though for the GE I'd say just "GE Plug-in Outdoor Switch", leave 'smart' out to make it shorter.

We should also add another fingerprint for the DZPA1: `mfr:"001D", prod:"3601", model:"0001"`. That's what Shilpa's is reporting for the MSR data, possibly an earlier run.